src/components/Onboarding/Onboarding.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import Button from 'cozy-ui/transpiled/react/Button'
+import Empty from 'cozy-ui/transpiled/react/Empty'
+import HomeCloud from 'src/assets/icons/HomeCloud.svg'
+
+import './styles.styl'
```

Comment:
We're used to name the import: 

```
import styles from ./styles.styl
...className={styles['onboarding']}
```

Like that, the bundle is able to know if the file is used or not. Without it should be marked as a side effect and bye bye tree shaking

Reply:
Ok,

But due to our last discussion on css vs utils, I removed this file and used only classnames 👍 

But I keep this rule in mind for future code